ui/app/routes/vault/cluster/secrets/backend/configuration/index.js

```diff
@@ -41,7 +51,7 @@ export default class SecretsBackendConfigurationRoute extends Route {
       }
     }
     // If the engine is configurable fetch the config model(s) for the engine and return it alongside the model
-    if (CONFIGURABLE_SECRET_ENGINES.includes(secretEngineModel.type)) {
+    if (CONFIGURABLE_SECRET_ENGINES.includes(secretEngineModel.type) || secretEngineModel.type === 'azure') {
```

Comment:
It looks like `azure` is included in `CONFIGURABLE_SECRET_ENGINES` so I think the `||` can be removed?

Reply:
Ah, good catch.